eclipse-scout-core/src/form/fields/groupbox/GroupBox.ts

```diff
@@ -424,13 +425,21 @@ export class GroupBox extends CompositeField implements GroupBoxModel {
       return;
     }
     this.$label = this.$title.appendDiv('label');
+    if (this._computeTitleVisible()) { // add it as a heading if its not the invisible main box
```

Comment:
Actually, the label can be change dynamically, but then we would have to recompute the levels for all headers, so we decided not to handle it, right?

Reply:
Correct